### DISCUSSION TITLE
Camera not listed in timelapse function

### DISCUSSION THREAD

**Discussion by brunodcassio** [Guilouz/Creality-Helper-Script-Wiki#525] [Issues]
![Captura de tela 2024-04-18 221324](https://github.com/Guilouz/Creality-K1-Series/assets/89844045/c0e0ff9c-fb83-48c8-87b5-2d552d34dbf5)

**Comment by Biospargel** [comment]
same here

**Comment by jaxxtheone** [comment]
If you press the little triangle next to the camera, you'll see the camera listed (if you already have this camera) as your default on Fluidd/Mainsail.

**Reply by brunodcassio** [reply]
![image](https://github.com/Guilouz/Creality-K1-Series/assets/89844045/932764bd-6ee5-4e03-8ea2-3b391f051dec)

**Reply by brunodcassio** [reply]
ela não está na lista, mas está funcionando como webcam. Já ativei ela também através do acesso root, onde ela aparece como uma câmera secundária na lista de webcam, mas mesmo assim não consta na lista de câmeras para o timelapse.

**Reply by brunodcassio** [reply]
![image](https://github.com/Guilouz/Creality-K1-Series/assets/89844045/609e6320-e07b-4295-8e79-d53abd57921a)

**Reply by brunodcassio** [reply]
me refiro a este menu. já ativei o item 4, mas não adiantou.

**Reply by jaxxtheone** [reply]
![FireShot Capture 049 -  54%  - fluidd K1 - Settings - 192 168 10 154](https://github.com/Guilouz/Creality-K1-Series/assets/26436482/840c29bf-43eb-49a4-b4ab-4e1bcf2a2076)
In this particular printer, I use Fluidd (not Mainsail). That's what I use as timelapse camera. K1 is the name od the camera and that's the same I use to have my timlapses..
I think you have to name it in the section Webcams you have in your installation. Once again, the name is the same, K1 in my installation.
